src/Core.ts

```diff
@@ -1064,14 +1071,21 @@ class Mvvm {
 
 			for (const name of el.getAttributeNames()) {
 				const expression: string = el.getAttribute(name);
-
-				if (name === "data-c-region") {
+				if (name === (ATTRIBUTE_PREFIX + "region")) {
```

Comment:
This introduces a string addition in the heap every time that "region" is concatenated.  This should be a separate constant the can be computed as a concatenation, but only once.

Reply:
Done.